client/src/app/components/login/login.component.ts

```diff
@@ -75,7 +77,7 @@ export class LoginComponent implements OnInit {
 
   resetForm(): void {
     this.loginForm.reset();
-    Object.keys(this.loginForm.controls).forEach(key => {
+    Object.keys(this.loginForm.controls).forEach((key) => {
       this.loginForm.controls[key].setErrors(null);
     });
```

Comment:
can we actually make this a single line instead?

For example:
```JavaScript
Object.keys(this.loginForm.controls).forEach(key => this.loginForm.controls[key].setErrors(null));
```

Reply:
I'm not sure i'm gona try, i didn't add that i think it was @makchamp .

Reply:
I will keep it this was actually. It is not a big deal for now.

Reply:
it's just a personal preference tbh, it doesn't change the functionality at all